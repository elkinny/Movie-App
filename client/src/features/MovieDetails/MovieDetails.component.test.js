import MovieDetailsComponent from './MovieDetails.component';
import React from 'react';
import { shallow } from './../../test/enzymeSetup.js';

describe('MovieDetailsComponent: ', () => {
  it('1. To match snapshot', () => {
    const props = {
      currentMovie: {
        duration: '120min',
        rating: '4.0',
        title: 'Captain Marvel',
        genre: 'Action',
        year: 2019,
        description:
          'Carol Danvers becomes one of the universe`s most powerful heroes when Earth is caught in the middle of a galactic war between two alien races.',
        link: 'https://www.youtube.com/watch?v=0LHxvxdRnYc',
        picLink:
          'https://m.media-amazon.com/images/M/MV5BMTE0YWFmOTMtYTU2ZS00ZTIxLWE3OTEtYTNiYzBkZjViZThiXkEyXkFqcGdeQXVyODMzMzQ4OTI@._V1_SY1000_CR0,0,675,1000_AL_.jpg',
      },
      movieCardRef: () => {},
      moviesByGenre: [],
    };
    const component = shallow(<MovieDetailsComponent {...props} />);
    expect(component).toMatchSnapshot();
  });
});
