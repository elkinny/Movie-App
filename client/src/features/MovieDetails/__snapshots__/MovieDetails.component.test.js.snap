// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetailsComponent:  1. To match snapshot 1`] = `
<Fragment>
  <MovieCardComponent
    description="Carol Danvers becomes one of the universe\`s most powerful heroes when Earth is caught in the middle of a galactic war between two alien races."
    duration="120min"
    genre="Action"
    link="https://www.youtube.com/watch?v=0LHxvxdRnYc"
    picLink="https://m.media-amazon.com/images/M/MV5BMTE0YWFmOTMtYTU2ZS00ZTIxLWE3OTEtYTNiYzBkZjViZThiXkEyXkFqcGdeQXVyODMzMzQ4OTI@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
    rating="4.0"
    title="Captain Marvel"
    year={2019}
  />
  <Navigation>
    <span>
      Films by 
      Action
       genre
    </span>
  </Navigation>
  <MoviesListComponent
    movies={Array []}
  />
</Fragment>
`;
